llvm/unittests/TargetParser/TripleTest.cpp

```diff
@@ -1341,6 +1341,12 @@ TEST(TripleTest, ParsedIDs) {
   EXPECT_TRUE(T.isTime64ABI());
   EXPECT_TRUE(T.isHardFloatABI());
 
+  T = Triple("x86_64-pc-linux-llvm");
+  EXPECT_EQ(Triple::x86_64, T.getArch());
+  EXPECT_EQ(Triple::PC, T.getVendor());
```

Comment:
Can we remove the `pc` part? :pray:

Reply:
It's just a test, doesn't affect anything.

Reply:
It's a waste of an assert. The only part we even care about is the environment check.

Reply:
Point is to make sure the whole triple gets parsed correctly AFAIK.